CHANGELOG.md

```diff
@@ -1,15 +1,21 @@
 ## Changes to the Mapbox Navigation SDK for iOS
 
-## Master
+## master
 
-#### User interface
-* Added support for abbreviated top banner instructions. [#1169](https://github.com/mapbox/mapbox-navigation-ios/pull/1169)
-* Reveal the steps list by swiping down on the top banner. [#1150](https://github.com/mapbox/mapbox-navigation-ios/pull/1150)
-* Fixed the poor contrast of the `TimeRemainingLabel.appearance().trafficSevereColor`, when navigation night style (theme) is enabled. [#1228](https://github.com/mapbox/mapbox-navigation-ios/pull/1228)
-* Data representing spoken voice instructions is now cached locally for reuse. [#1229](https://github.com/mapbox/mapbox-navigation-ios/pull/1229)
+### User interface
+* `DayStyle` is temporarily replaced with `NightStyle` while the user travels through a tunnel. ([#1127](https://github.com/mapbox/mapbox-navigation-ios/pull/1127))
```

Comment:
This is not entirely correct. Styles with `styleType` set to `.night` is temporarily applied while the user travels through a tunnel.